### PR TITLE
feat: data_generator for random phone numbers

### PR DESCRIPTION
**Pull request by jaimemrjm**
`RANDOM_PHONE_NUMBER` variable replacement with country support based on Faker library.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,5 @@ selenium>=2.53.6,<4              # web tests
 Appium-Python-Client>=0.24,<=1.3 # mobile tests
 screeninfo==0.3.1
 lxml==4.*
+Faker==4.14.0
```
Comment (rgonalo) [inline requirements.txt:6]: latest Faker version is 13.0.0, can it be used?

File: toolium/utils/data_generator.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Telefónica Digital.
+# CDO QA Team <qacdo@telefonica.com>
```
Comment (rgonalo) [inline toolium/utils/data_generator.py:4]: Toolium Apache license header must be used

Comment (jaimemrjm) [reply toolium/utils/data_generator.py]: OK, fixed.

**Review by rgonalo — DISMISSED** [review]

**Comment by rgonalo** [comment]
Stale PR without activity in the last nine months

**Pull request closed without merging**